packages/bpk-component-select/src/BpkSelect.scss

```diff
@@ -104,6 +104,7 @@ $select-image-large-height: 22 * $bpk-one-pixel-rem;
     left: $bpk-spacing-sm;
     width: $select-image-width;
     height: $select-image-height;
+    pointer-events: none;
```

Comment:
This addresses https://gojira.skyscanner.net/browse/WOM-557, right?

Reply:
Yes that is correct